src/ray/protobuf/gcs_service.proto

```diff
@@ -177,6 +177,13 @@ service ActorInfoGcsService {
   rpc KillActorViaGcs(KillActorViaGcsRequest) returns (KillActorViaGcsReply);
 }
 
+message GetClusterIdRequest {}
+
+message GetClusterIdReply {
+  GcsStatus status = 1;
```

Comment:
Do we need this?

Reply:
No, but it seems to be convention in other RPC replies, even when it's not used.